src/EditorFeatures/CSharpTest/Completion/CompletionProviders/ExplicitInterfaceMemberCompletionProviderTests.cs

```diff
@@ -1044,13 +1044,12 @@ interface I
 
             class C : I
             {
-                void I.M(int x = 10)
+                void I.M(int x)
                 {
                     throw new System.NotImplementedException();
                 }
             }
             """;
-        // TODO: Consider adding the default value too.
```

Comment:
Since default values are meaningless in explicit interface implementations, showing them in the completion item is also unimportant. Removed this comment to reflect the sanity of the current case.